torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19497,6 +19497,14 @@ def __init__(
         "_refs.nn.functional.elu",
         torch_opinfo_name="nn.functional.elu",
     ),
+    PythonRefInfo(
```

Comment:
How good are the sample inputs for hardtanh? Do they validate the legacy hack works as expected?

Reply:
uhh... seem quite bad actually - it looks like they don't even test non-default values of `min_val` and `max_val`? https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_methods_invocations.py#L7040

I'll update them and add an ErrorOpInfo if we decide to preserve the current behavior.

Reply:
Instead of updating the samples you can add reference inputs instead, which can be more extensive and won't affect existing tests. See 

https://github.com/pytorch/pytorch/blob/b651148fc305491c75dbba52d6bccf76b2ec114b/torch/testing/_internal/common_methods_invocations.py#L3815

for an example of what these functions look like